index.html

```diff
@@ -16,12 +16,100 @@
+            <h1 class="text text3">Projects</h1>
             <p class="text text2">Hello! I am a software developer! I can help you build a product, feature or website. Taka a look of my works. If you like what you see and have a project you need coded, don’t hesitate and contact me.</p>
         </section>
         <section class="rectangle">
             <h4>Start collaboration</h4>
         </section>
         <section class="secondimagecontain"></section>
+        <section class="projectpageimage1"></section>
+        <section class="rectangle1">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container">
+            <section class="grid-item box1">HTML/CSS</section>
+            <section class="grid-item box2">Ruby on Rails</section>
+            <section class="grid-item box3">Javascript</section>
+        </section>
+        <section class="rectangle2">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
+        <section class="projectpageimage2"></section>
+        <section class="rectangle3">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container1">
+            <section class="grid-item1 box4">HTML/CSS</section>
+            <section class="grid-item1 box5">Ruby on Rails</section>
+            <section class="grid-item1 box6">Javascript</section>
+        </section>
+        <section class="rectangle4">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
+        <section class="projectpageimage3"></section>
+        <section class="rectangle5">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container2">
+            <section class="grid-item1 box7">HTML/CSS</section>
+            <section class="grid-item1 box8">Ruby on Rails</section>
+            <section class="grid-item1 box9">Javascript</section>
+        </section>
+        <section class="rectangle6">
+            <h4 class="text text5">See this project</h4>
+            <img class="img img3" src="arrowsymbol.svg" alt="arrow symbol for webpage">
+        </section>
+        <section class="projectpageimage4"></section>
+        <section class="rectangle7">
+            <h2 class="text text4">Project name goes here</h2>
+        </section>
+        <section class="grid-container2">
+            <section class="grid-item1 box10">HTML/CSS</section>
+            <section class="grid-item1 box11">Ruby on Rails</section>
+            <section class="grid-item1 box12">Javascript</section>
+        </section>
+        <section class="rectangle8">
+            <h4 class="text text5">See this project</h4>
```

Comment:
- [ ] Please try to add the  `see project` as a link a or button tag, and follow the button styling: Active, Hover and Pressed